app/notifiers/notifier/base.rb

```diff
@@ -0,0 +1,72 @@
+# The base class of notifiers. This is meant to be called by the Notifications Framework
+#
+# @api notifications
+class Notifier::Base
+  include ApplicationNotificationsConcern
+
+  class << self
+    # This is to allow client code to call create notifications without explicitly creating
```

Comment:
"This is to allow client code to create notifications without explicitly instantiating notifiers"